benchexec/tablegenerator/columns.py

```diff
@@ -132,32 +130,37 @@ def __init__(
         relevant_for_diff=None,
         display_title=None,
     ):
-        # If scaling on the variables is performed, a display unit must be defined, explicitly
-        if scale_factor is not None and scale_factor != 1 and unit is None:
-            raise util.TableDefinitionError(
-                f"Scale factor is defined, but display unit is not (in column {title})"
-            )
+        with decimal.localcontext(local_context):
+            assert (
+                decimal.getcontext().rounding == decimal.ROUND_HALF_UP
+            ), f"rounding of context is {decimal.getcontext().rounding}, expected ROUND_HALF_UP"
```

Comment:
These assertions are not needed anymore once we properly use a local context. We do not fear anyone accidentally changing our context, we fear that people forget to use `localcontext` where necessary, but the assertions don't help with that.